packages/google_sign_in/google_sign_in_ios/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_sign_in_ios
 description: iOS implementation of the google_sign_in plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/google_sign_in/google_sign_in_ios
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+google_sign_in%22
-version: 5.5.1
+version: 5.5.2
```

Comment:
Is this one really worth a version bump?

Reply:
I see @GaryQian's similar comment now.  It seems like the ones that are just adding a lint ignore aren't worth it, but up to you.

Reply:
I'll go back and scrub them since being lazy and just using the tool defaults is clearly causing more confusion than it's saving.